src/common/utils.ts

```diff
@@ -256,7 +256,15 @@ export const gucci = async (tmpl: string, args: { [key: string]: string }): Prom
   const gucciArgs = Object.entries(args).map(([k, v]) => `-s ${k}='${v ?? ''}'`)
   const quoteBackup = $.quote
   $.quote = (v) => v
-  const processOutput = await nothrow($`gucci ${gucciArgs} ${tmpl}`)
+  let processOutput
+  if (existsSync(tmpl)) {
+    // input string is a file path
+    processOutput = await nothrow($`gucci ${gucciArgs} ${tmpl}`)
+  } else {
+    // input string is a go template content
+    const str = tmpl.replace(/"/g, '\\"')
```

Comment:
You can use `.replaceAll('"','\"') doesn't need regular expression

Reply:
didn't work.

Reply:
What do you mean? What didn't work? Please elaborate